pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -913,18 +923,29 @@ func configureLocalDiskToMigrate(dom *libvirtxml.Domain, vmi *v1.VirtualMachineI
 		} else {
 			dom.Devices.Disks[i].Source.Slices.Slices = append(dom.Devices.Disks[i].Source.Slices.Slices, slice)
 		}
+		var path string
+		_, hotplugVol := hotplugVols[name]
+		if hotplugVol {
+			path = hotplugdisk.GetVolumeMountDir(name)
+		}
 		// Adjust the XML configuration when it migrates from a filesystem source to a block destination or vice versa
 		if _, ok := fsSrcBlockDstVols[name]; ok {
 			log.Log.V(2).Infof("Replace filesystem source with block destination for volume %s", name)
+			if !hotplugVol {
+				path = filepath.Join(string(filepath.Separator), "dev", name)
+			}
 			dom.Devices.Disks[i].Source.Block = &libvirtxml.DomainDiskSourceBlock{
-				Dev: filepath.Join(string(filepath.Separator), "dev", name),
+				Dev: path,
```

Comment:
In hotplug case this is a directory. How does that work?

And how did this ever work with "dev"?